### DISCUSSION TITLE
Blizzballer sorc seems to be the fastest for leveling - by far

### DISCUSSION THREAD

**Discussion by zylo-** [isid0re/SoloLeveling#141] [General]
I started 3 sorcs lvl 1 at exact same time and now:
Blizzballer is level 44
Blizzard sorc is lvl 33
Meteorb sorc is lvl 32

I know these depend on a lot of factors such as luck, but the Blizzballer seems much further ahead.

Just FYI.

**Comment by SebastianDpunkt** [comment]
idk what I'm doing wrong. My Blizzard sorc won't attack at all after respec.

MeteOrb was fine; Druid, Necro and now blizzard all lay down killing stuff after respec.

**Comment by JediPeaceFrog** [comment]
can you point me to where the instructions are to start 3 chars at same time?  thanks.

**Comment by zylo-** [comment]
Just name them diff profiles
Add
1 2 3 at the end etc
________________________________
